### ISSUE TITLE
Join working groups as part of on-boarding

### ISSUE THREAD

**Issue by cassiezamparini** [openedx/wg-coordination#141]
[See proposal](https://openedx.atlassian.net/wiki/spaces/COMM/pages/4522278925/Join+working+groups+as+part+of+on-boarding)

**Comment by jalondonot** [comment]
@navinkarkera, during the [last meeting of the Coordination Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/4643323905/2024-11-26+CC+Working+Group+Meeting+Notes), we agreed that creating a work plan for each CC Enhancement initiative would be beneficial. We understand that progress may be challenging due to the Christmas and New Year's holidays. However, when you resume work in January (I guess), we would greatly appreciate it if you could share the next steps to advance your initiative. This will help engage other members of the Open edX community who may be interested in supporting your efforts. Please let us know once you have a timeline in mind so that we can assist you in any way possible. Thanks and Merry Christmas!

**Comment by antoviaque** [comment]
@navinkarkera Same here, could you reply to the question above?

**Comment by navinkarkera** [comment]
@antoviaque Thanks for the ping!

@jalondonot Apologies for the delay in response. I updated the instructions for CC participants and administrators here:

* [Coding CC participants instructions](https://openedx.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=3358458250&selectedPageVersions=20&selectedPageVersions=18)
* [Non-coding CC participants instructions](https://openedx.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=3359113457&selectedPageVersions=15&selectedPageVersions=13)
* [Admin instructions](https://openedx.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=2810249408&selectedPageVersions=82&selectedPageVersions=81)

**Comment by antoviaque** [comment]
@navinkarkera Great, thank you! Does that complete the current task, or are there other areas where we should direct new core contributors towards the working groups? Maybe in the onboarding courses, too? The handbook will also likely reference this, though it is less focused on onboarding than the courses (cc @cassiezamparini )

**Comment by navinkarkera** [comment]
@antoviaque Yes, the original plan of updating relevant documentation pages is complete. But I do like the idea of including something in onboarding course.

**Comment by antoviaque** [comment]
@navinkarkera Great! Should we keep tracking it in the current task, ie is this something you are interested in contributing too?

**Comment by navinkarkera** [comment]
@antoviaque ~Sure, we can track it here. I have created an account in the training site, who should I contact to get staff access?~

I just checked the course contents, it already has a section encouraging core contributors to join working groups. Sorry for not checking it beforehand 😅

**Comment by antoviaque** [comment]
@navinkarkera All good then! Thanks for checking - I will thus mark this ticket as done 👍
